### PR TITLE
test(account-lib): add missing test

### PR DESCRIPTION
**Pull request by facundoCarreSimtlix**
add missing test for valid address

stlx-13732

### CODE REVIEW

**Review by CapnMigraine — CHANGES_REQUESTED** [review]
Please make this suggested change also
https://github.com/BitGo/BitGoJS/pull/2032/files#r810299252

**Review by CapnMigraine — DISMISSED** [review]

File: modules/account-lib/src/coin/near/utils.ts [hunk 1]
```diff
@@ -25,9 +25,9 @@ export class Utils implements BaseUtils {
 
   /** @inheritdoc */
   isValidPublicKey(key: string): boolean {
-    const accountIDAddressRegex = '^(([a-z\\d]+[\\-_])*[a-z\\d]+\\.)*([a-z\\d]+[\\-_])*[a-z\\d]+$';
     try {
-      new RegExp(accountIDAddressRegex).test(key) || new KeyPair({ pub: key });
+      // this regex is from near doc, https://docs.near.org/docs/concepts/account
+      /^(([a-z\d]+[\-_])*[a-z\d]+\.)*([a-z\d]+[\-_])*[a-z\d]+$/.test(key) || new KeyPair({ pub: key });
```
Comment (ewangbitgo) [inline modules/account-lib/src/coin/near/utils.ts:30]: is there a length restriction of address?

Comment (facundoCarreSimtlix) [reply modules/account-lib/src/coin/near/utils.ts:30]: image
i found this in the doc, i can add this restriction in this pr

Comment (CapnMigraine) [reply modules/account-lib/src/coin/near/utils.ts:30]: Yes @facundoCarreSimtlix, implicit accounts have 64 characters, but account ids can have between 2 and 64. Please add that validation to the regex pattern

**Review by CapnMigraine — APPROVED** [review]

**Review by ewangbitgo — APPROVED** [review]

**Pull request merged**